src/tests/ftest/launch.py

```diff
@@ -108,8 +108,8 @@ def set_test_environment():
         os.path.abspath("util/apricot"),
         os.path.abspath("util"),
         os.path.abspath("../../utils/py"),
-        os.path.join(base_dir, "lib", "python2.7", "site-packages"),
-        os.path.join(base_dir, "lib", "python3", "site-packages"),
+        os.path.join(base_dir, "lib64", "python2.7", "site-packages"),
+        os.path.join(base_dir, "lib64", "python3", "site-packages"),
```

Comment:
Always dding both of these paths to the `PYTHONPATH` feels dangerous.  If the same module is available in both, won't the 2.7 module get used even when using Python 3?